web_external/templates/body/challengeLearn.jade

```diff
@@ -133,14 +133,14 @@
             platform--an open source data and analytics platform made up
             of three major components:
             <a href="http://tangelo.readthedocs.org">Tangelo</a>,
-            <a href="http://girder.readthedocs.org">Girder</a>, and
-            <a href="http://romanesco.readthedocs.org">Romanesco</a>.
+            <a href="http://girder.readthedocs.org">Girder</a>, and the
+            <a href="https://github.com/girder/girder_worker">Girder worker</a>.
```

Comment:
'Worker' should be capitalized here.
Reference: https://github.com/girder/girder_worker/commit/15283b2a810d1872322049002531fff53927c1b0